draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -805,10 +805,10 @@ The Issuer creates Disclosures first for the sub-claims and then includes their
 Upon receiving an SD-JWT, either directly or as a component of an SD-JWT+KB, a Holder
 or a Verifier needs to ensure that:
 
- * the Issuer-signed JWT is valid, i.e., it is signed by the Issuer, the signature is valid, it is not expired, it is not suspended or revoked, etc., and
+ * the Issuer-signed JWT is valid, and
  * all Disclosures are valid and correspond to a respective digest value in the Issuer-signed JWT (directly in the payload or recursively included in the contents of other Disclosures).
 
-The Holder or the Verifier MUST perform the following (or equivalent) steps when receiving
+The Holder or the Verifier MUST perform the following steps when receiving
```

Comment:
I still think some kind of allowance to reasonably allow for different implementation choices like order of some steps or use of different intermediate data structures than what's implied.

Reply:
```suggestion
The Holder or the Verifier MUST perform the following checks when receiving
```